packages/react-components/react-tree/stories/Tree/flattenTreeFromElements.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import {
+  Tree,
+  TreeItem,
+  TreeItemLayout,
+  useFlatTreeItems_unstable,
+  flattenTreeFromElements_unstable,
+} from '@fluentui/react-tree';
+
+const defaultItems = flattenTreeFromElements_unstable(
```

Comment:
is there a requirement to parse JSX children?

Reply:
Nope, it's just a nice to have